src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    public static final String SHAPE_NAME = "Right Triangle";
```

Comment:
Why this constant are public if it's usage limited by scope of this class? (other triable as well)

Reply:
Agree. Thanks to IDEA :-)
Fixed.